app/uk/gov/nationalarchives/omega/editorial/controllers/EditSetController.scala

```diff
@@ -145,11 +163,48 @@ class EditSetController @Inject() (
   def view(id: String): Action[AnyContent] = Action { implicit request: Request[AnyContent] =>
     withUser { user =>
       logger.info(s"The edit set id is $id ")
-      val editSetModels = editSets.getEditSet()
-      val messages: Messages = request.messages
-      val title: String = messages("edit-set.title")
-      val heading: String = messages("edit-set.heading", editSetModels.name)
-      Ok(editSet(user, title, heading, editSetModels))
+      generateEditSetView(user, fallbackEditSetReorder)
+    }
+  }
+
+  def viewAfterReordering(idCurrentlyUnused: String): Action[AnyContent] = Action {
```

Comment:
Can we log `idCurrentlyUnused` like in the `view` controller, if just to not have the warning?

Reply:
Sure. Will change.

Reply:
Changed.